apps/src/templates/courseRollupPages/RollupLessonEntry.jsx

```diff
@@ -18,12 +17,7 @@ export default class RollupLessonEntry extends Component {
       <div style={styles.main}>
         <div style={styles.header}>
           <a href={this.props.lesson.link} style={styles.link}>
-            <h3>
-              {i18n.lessonNumbered({
-                lessonNumber: this.props.lesson.position,
-                lessonName: this.props.lesson.displayName,
-              })}
-            </h3>
+            <h3>{this.props.lesson.title}</h3>
```

Comment:
lesson.title is based on `lesson.localized_title` which uses the lesson's `relative_position`. in each case where I switched to using `lesson.title`, I verified that the original position value being supplied to `i18n.lessonNumbered` was also the `relative_position`, as opposed to the `absolute_position` (or some other computed value).